Source/WebKit/WebProcess/Network/webrtc/LibWebRTCNetworkManager.h

```diff
@@ -32,16 +32,20 @@
 #include <WebCore/ProcessQualified.h>
 #include <WebCore/RTCNetworkManager.h>
 #include <WebCore/ScriptExecutionContextIdentifier.h>
+#include <wtf/RefCounted.h>
 #include <wtf/TZoneMalloc.h>
 
 namespace WebKit {
 
-class LibWebRTCNetworkManager final : public WebCore::RTCNetworkManager, public rtc::NetworkManagerBase, public webrtc::MdnsResponderInterface, public WebRTCMonitorObserver {
+class LibWebRTCNetworkManager final : public RefCounted<LibWebRTCNetworkManager>, public WebCore::RTCNetworkManager, public rtc::NetworkManagerBase, public webrtc::MdnsResponderInterface, public WebRTCMonitorObserver {
     WTF_MAKE_TZONE_ALLOCATED(LibWebRTCNetworkManager);
 public:
-    static LibWebRTCNetworkManager* getOrCreate(WebCore::ScriptExecutionContextIdentifier);
+    static RefPtr<LibWebRTCNetworkManager> getOrCreate(WebCore::ScriptExecutionContextIdentifier);
     ~LibWebRTCNetworkManager();
 
+    void ref() const final { RefCounted::ref(); }
```

Comment:
Use DEFINE_VIRTUAL_REFCOUNTED macro?

Reply:
I like being explicit about it. I find the macros confusing.